binding.gyp

```diff
@@ -74,6 +74,17 @@
           },
         }]
       ]
+    },
+    {
+      "target_name": "action_after_build",
+      "type": "none",
+      "dependencies": [ "<(module_name)" ],
+      "copies": [
+          {
+            "files": [ "<(PRODUCT_DIR)/<(module_name).node" ],
```

Comment:
if you want to include xmljs.pdb, we could add that to copy here

I don't think the other build artifacts are necessary, node certainly doesn't use anything but xmljs.node